CONTRIBUTING.md

```diff
@@ -35,3 +35,26 @@ This subfolder contains some example `.rst` files that show how to implement spe
 2. Run `pip install -r requirements-dev.txt`
 3. Run `pip install .` (must do this every time there is a change to the theme).
 4. Run `sphinx-build -b html docs/ docs/_build/html` to build the documentation. This will create the `/docs/_build` subfolder where you can see the html files generated based on the `.rst` files in the `docs` folder and the styles pulled from the `qiskit_sphinx_theme`.
+
+------
+## Releases
+
+1. Bump the version:
+   1. `git pull main`
+   2. Create a new branch
+   3. Bump `setup.py` and `qiskit_sphinx_theme/__init__.py` to use the new version
+   4. PR the change and land it
+2. Push the Git tag:
+   1. `git pull main` to pull the version bump. If other commits have landed since the version bump, use `git revert --hard <sha>` to change to the version bump.
+   2. `git tag <version>`, e.g. 1.11.0
+   3. `git push upstream <version>`
+3. Check that the release worked:
+   1. Check that the tag shows up in https://github.com/Qiskit/qiskit_sphinx_theme/tags
+   2. The pip release is automated with [GitHub Actions](https://github.com/Qiskit/qiskit_sphinx_theme/actions). After a few minutes, check that https://pypi.org/project/qiskit-sphinx-theme/#history has the release. (You can skip to the next step while waiting)
+4. Announce the release on GitHub:
+   1. On https://github.com/Qiskit/qiskit_sphinx_theme/tags, click the `...` to the right of the released tag's row. Click "Create release"
+   2. Add release notes
+      1. Add the sections `**Features / API Changes:**` and `**Bug Fixes:**`. 
+      2. Use `git diff` to see what changes have been made. Copy and paste those entries as bullets into the relevant sections. Ignore any "internal only" changes like CI changes or updates to the README.
```

Comment:
The output of `git diff` is too noisy, don't you think? The Github compare/ feature seems to look great: `https://github.com/Qiskit/qiskit_sphinx_theme/compare/<previous release tag>...<new release tag>` for example: https://github.com/Qiskit/qiskit_sphinx_theme/compare/1.10.2...1.11.0rc1

Reply:
Good catch! I meant to say `git log --oneline`.